README.md

```diff
@@ -232,15 +232,24 @@ For a 52B model. See full run configs in `MaxText/configs/` as `1xv4-384.sh` and
 
 For a 16B model. See full run configs in `MaxText/configs/` as `16b.sh`.
```

Comment:
can you add the 64B and 128B results as well?

Reply:
Generally, we want a sensible getting started page for users that show our full results. I think in the main README we should just say we get between 55% MFU and 67% MFU between 256 and 4096 v5e chips and 16B to 128B params. Then we should link to the v5e results MD page.

Reply:
Done. Added the 64B and 128B results and updated the v5e results md page.